python/dcr_utils.py

```diff
@@ -126,9 +220,9 @@ def fft_shift_convolve(image, shift, n_substep=100, useInverse=False, weights=No
     """
     y_size, x_size = image.shape
     kernel_x = kernel_1d(shift.dx, x_size, n_substep=n_substep, debug_sinc=False,
-                         useInverse=useInverse, weights=weights)
+                         useInverse=useInverse, weights=weights, lanczos=4)
```

Comment:
Why was 4 found to be appropriate order and do you want to make this a defined constant elsewhere?